deploy/runtools/run_farm.py

```diff
@@ -378,6 +378,34 @@ def fpga_node_xdma(self):
                 run('make clean')
                 run('make')
 
+    def fpga_node_qcow(self):
+        """ Install qemu-img management tools and copy NBD infra to remote
+        node. This assumes that the kernel module was already built and exists
+        in the directory on this machine.
+        """
+        self.instance_logger("""Setting up remote node for qcow2 disk images.""")
+        with StreamLogger('stdout'), StreamLogger('stderr'):
+            # get qemu-nbd
+            run('sudo yum -y install qemu-img')
+            # copy over kernel module
+            put('../build/nbd.ko', '/home/centos/nbd.ko', mirror_local_mode=True)
+
+    def load_nbd_module(self):
+        """ load the nbd module. """
+        self.unload_nbd_module()
```

Comment:
I think it might be good to capture in comments why we have to unload first>

Reply:
done